### PR TITLE
switch to dune

### PR DESCRIPTION
**Pull request by redianthus**
Hi,

I switched the project to dune. The library build and tests are working.

### CODE REVIEW

File: xdg-basedir.opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "ocaml"
+  "fileutils"
+  "unix"
+]
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+description: "This library implements the xdg-basedir specification. It helps to define standard locations for configuration, cache and data files in the user directory and on the system. It is a straightforward implementation on UNIX platform and try to apply consistent policies with regard to Windows directories. It is inspired by the Haskell implementation of this specification, and it follows the same choices for Windows directories. [The xdg-basedir specification](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html) and [the Haskell implementation](http://github.com/willdonnelly/xdg-basedir) and [the API of this implementation](http://xdg-basedir.forge.ocamlcore.org/api)."
```
Comment (gildor478) [inline xdg-basedir.opam]: Can you split the description to be multiple lines. A very long line is hard to read.

File: _headache.config [hunk 2]
```diff
@@ -1,67 +0,0 @@
-################################################################################
```
Comment (gildor478) [inline _headache.config]: Please keep.

File: _header [hunk 3]
```diff
@@ -1,18 +0,0 @@
-xdg-basedir: XDG basedir location for data/cache/configuration files
```
Comment (gildor478) [inline _header]: Please keep.

**Review by gildor478 — COMMENTED** [review]
Please review my comments.

File: _headache.config (hunk 2 above)
Comment (redianthus) [reply _headache.config]: Will do, but I'm curious, why would you want to keep this one and the other one ?

Comment (gildor478) [reply _headache.config]: They are configuration files for headache, I use them to update the header of files from time to time.

**Comment by redianthus** [comment]
Done. :)

**Comment by redianthus** [comment]
Thanks ! Are you planning a release on opam ? Or do you want me to update it there directly ?

**Comment by gildor478** [comment]
I am not planning to do the release on OPAM anytime soon, that would be awesome if you can do it.

**Comment by redianthus** [comment]
@gildor478, I'm going to do it. Could you release `0.0.4` so I can update it on OPAM ?

**Comment by gildor478** [comment]
@zapashcanon 
Sorry for the delay, I had to fix a couple of missing bits.

I have published the new release directly on GitHub.
https://github.com/gildor478/ocaml-xdg-basedir/releases/tag/0.0.4

Use the .tar.gz from this link rather than the source code provided by default. I have signed the .tar.gz, it might be useful to someone who wants to verify the archive.

Thanks

**Pull request merged**